lib/hammer_cli_foreman/table_preferences.rb

```diff
@@ -0,0 +1,71 @@
+module HammerCLIForeman
+  class TablePreferences < HammerCLIForeman::Command
+    resource :table_preferences, :users
+    command_name "table-preferences"
```

Comment:
Let's stick with our conventions:
```suggestion
    command_name "table-preference"
```